emission/tests/netTests/TestStats.py

```diff
@@ -35,30 +113,54 @@ def testCreateEntry(self):
   def testSetClientMeasurements(self):
     currTime = time.time()
     stats.setClientMeasurements("testUser", self.testInputJSON)
-    for savedEntry in get_client_stats_db().find():
-      self.assertEquals(savedEntry['client_app_version'], '2.0.1')
-      self.assertEquals(savedEntry['client_os_version'], '4.3')
-      self.assertAlmostEqual(savedEntry['reported_ts'], time.time(), places = 0)
-      if savedEntry['stat'] == 'sync_pull_list_size':
-        self.assertIn(savedEntry['ts'], [1411418998701, 1411418998702, 1411418998703])
-        self.assertIn(savedEntry['reading'], [1111, 2222, 3333])
+    savedEntries = get_client_stats_db().query_tags()
+    self.assertEquals(type(savedEntries), list)
+    self.assertEquals(len(savedEntries), 2)
+
+
+    # @TODO: Improve this test to be more informative
+    for savedEntry in savedEntries:
+      #print(savedEntry)
+      self.assertEquals(savedEntry['Metadata']['client_app_version'], '2.0.1')
+      self.assertEquals(savedEntry['Metadata']['client_os_version'], '4.3')
+      #self.assertAlmostEqual(savedEntry['Metadata']['reported_ts'], time.time(), places = 0)
```

Comment:
I'm still working on figuring out how to make the queries. But I am under the impression that the plotter application gives us these queries for free.